### PR TITLE
BUG: Enable submitting jobs to compute instances

### LINKED ISSUE #901: Submitting a job to a compute instance fails because the number of nodes can't be read
compute instances don't have a flag "scale_settings".

### PR DESCRIPTION
**Pull request by ant0nsc**
This PR removes the check for available number of nodes, meaning the code will run just fine on any type of compute.
Closes #901

### CODE REVIEW

File: hi-ml-azure/testazure/testazure/test_himl.py [hunk 1]
```diff
@@ -172,94 +171,19 @@ def mock_workspace(mock_compute_cluster: MagicMock, dummy_compute_cluster_name:
...
-def test_validate_compute_real(tmp_path: Path) -> None:
+def test_validate_compute(tmp_path: Path) -> None:
     """
-    Get a real Workspace object and attempt to validate a compute cluster from it, if any exist.
-    This checks that the properties/ methods in the Azure ML SDK are consistent with those used in our
-    codebase
+    Get a real Workspace object and attempt to validate a compute cluster from it.
     """
     with check_config_json(tmp_path, shared_config_json=get_shared_config_json()):
         workspace = get_workspace(aml_workspace=None, workspace_config_path=tmp_path / WORKSPACE_CONFIG_JSON)
```
Comment (kenza-bouzid) [inline hi-ml-azure/testazure/testazure/test_himl.py:179]: lines 178-179 can be changed by 
from testhiml.utils_testhiml import DEFAULT_WORKSPACE
DEFAULT_WORKSPACE.workspace

**Review by kenza-bouzid — APPROVED** [review]
LGTM!

File: hi-ml-cpath/requirements_run.txt [hunk 2]
```diff
@@ -17,5 +17,6 @@ tifffile==2022.10.10
 timm==0.6.12
 torch==1.11
 torchmetrics==0.10.2
+torchstain==1.3.0
```
Comment (Shruthi42) [inline hi-ml-cpath/requirements_run.txt:20]: What is this library?

**Review by Shruthi42 — APPROVED** [review]

**Review by fepegar — APPROVED** [review]
LGTM. My only suggestion would be to replace "cluster" with "target" pretty much everywhere.

File: hi-ml-cpath/requirements_run.txt (hunk 2 above)
Comment (ant0nsc) [reply hi-ml-cpath/requirements_run.txt:20]: This was a library that was missing in the cpath run requirements and was needed to make the PR build pass.

**Pull request merged**